torch/ao/quantization/fx/prepare.py

```diff
@@ -420,7 +490,9 @@ def _get_target_activation_dtype_for_node(
         }
 
     elif node.op == 'output':
-        if outputs_seen_counter in output_quantized_idxs:
+        # Note: creating placeholder observer here is temporary, it will be moved
+        # to the new programmable API when that is ready
```

Comment:
I think I'm still confused on the "programmable" part fo the programmable API.  I thought the "programmable" part would be enabling the user to do something they cannot do with simple defaults because their modeling code is complicated.  Not sure why this specific case would be related.  I think the user should be able to override this in a programmable way, but the default shouldn't have to go through the programmable way.

Reply:
for L497:498, can that read from qconfig.py too instead of creating new hardcoded observer objects?

Reply:
this comment is saying we should not be creating new placeholder observer configs in prepare, this is just temporary

sure I can move L497:498 to qconfig.py as well

Reply:
> I think the user should be able to override this in a programmable way, but the default shouldn't have to go through the programmable way.

not sure what this means, in the end both programmable api or config api should end up with configuring the same thing, which is something like:
node.meta["input_act_obs_or_fq_ctr"] = ...
node.meta["output_act_obs_or_fq_ctr"] = ...
I think.

Reply:
how about something like:
* a reasonable default is specified in a configuration
* user can override things in a programmable way

Both of those things can live under the `QTaggerFutureNameTBD`, but in different places of it.

Reply:
yeah that is the plan, let's discuss in details when I put up that PR, current PR is still walking towards the goal of separating the configuration related things to a separate class